### PR TITLE
chore: Add trailing slashes to links in blog item page

### PR DESCRIPTION
**Pull request by insmac**

### CODE REVIEW

**Review by bluestreak01 — CHANGES_REQUESTED** [review]
image

**Review by bluestreak01 — CHANGES_REQUESTED** [review]
image

File: src/utils/addTrailingSlashToPath.ts [hunk 1]
```diff
@@ -0,0 +1,6 @@
+export const addTrailingSlashToPath = (path: string) => {
```
Comment (argshook) [inline src/utils/addTrailingSlashToPath.ts:1]: perhaps let's extract this to a reusable function?

https://github.com/questdb/questdb.io/blob/05c824f671a2127b21820bfe025cc68dd0d51ca3/src/theme/DocSidebar/index.tsx#L168-L169

File: src/theme/BlogPostPage/index.tsx [hunk 2]
```diff
@@ -7,12 +7,25 @@ import BlogSidebar from "@theme/BlogSidebar"
...
+    ;((prevItem.permalink as any) as string) = addTrailingSlashToPath(
+      prevItem.permalink,
+    )
+  }
+
+  if (nextItem != null) {
+    ;((nextItem.permalink as any) as string) = addTrailingSlashToPath(
+      nextItem.permalink,
+    )
+  }
```
Comment (argshook) [inline src/theme/BlogPostPage/index.tsx:27]: maybe it's possible to set types separately, in order to avoid the `as any as string`?

**Review by argshook — CHANGES_REQUESTED** [review]

**Review by argshook — APPROVED** [review]

**Comment by insmac** [comment]
@bluestreak01 Please re-check - I have fixed Blog main page and Blog individual pages.

**Review by bluestreak01 — APPROVED** [review]

**Pull request merged**